README.md

```diff
@@ -1,17 +1,192 @@
-2. All group members should then clone down the forked repository (make sure that everyone is added as a collaborator as well). Since you don't want your project to be named "whats-cookin-starter-kit", add an optional argument after the repo url when cloning. The command should look like this: `git clone [remote-address] [what you want to name the repo]`.
-3. Once you have cloned the repo, change into the directory and install the project dependencies. Run `npm install` or `npm i` to install project dependencies.
-4. Run npm start in the terminal to see the HTML page (you should see some boilerplate HTML displayed on the page). `Control + C` is the command to stop running the local server. Closing the terminal without stopping the server first could allow the server to continue to run in the background and cause problems. This command is not specific to Webpack; make note of it for future use.
-5. Make sure both members of your team are collaborators on the forked repo.
-6. Do not run `npm audit fix --force`. This will update to the latest version of packages. We need to be using `webpack-dev-server@3.11.2` which is not the latest version. If you start to run into Webpack errors, first check that all group members are using the correct version.
 
-## Testing
+<!-- ABOUT THE PROJECT -->
+## What's Cookin?!
 
-Mocha and chai are already set up, with a boilerplate test for you.
-# iron-fe
+### About The Project
+
+[![What's Cookin?!](https://user-images.githubusercontent.com/43621994/161454532-c8cd921c-c249-4021-8e18-b9d73df796e6.png)
+
+What's Cooking? Its a question we all have had at some point or another, and its a question we were tasked to solve 
+by Turing School of Software and Design. We were given a week to build a fully functional app using a given repo, 
+combining HTML, Css, and Javascript to create a well built user experience.
+
+With this application you are able to view a list of curated recipes and save them for future use. Users 
+will also be able to add these recipes to a Cooking list, Which in further versions of the application 
+they can use to do meal planning for their forseeable future.
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+### Built With
+
+In this project, we used these tools to help build, maintain, and populate the Web Application
+We were tasked with utilizing data from an API as well as using Webpack for the first time in the Turing program.
+
+#### Languages
+* Javascript
+* HTML
+* Css
+
+#### Frameworks/Libraries
+* [JQuery](https://jquery.com)
+* [Webpack](https://webpack.js.org/)
+* Whats Cookin Starter Kit Api as provided by Turing 
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+To start with, clone down the Repo on to your local machine. 
+Then, run NPM install in the console inside of that repository using this code.
+
+  npm install 
```

Comment:
Add backticks with install code and place under an install instruction subheader